ddtrace/contrib/tornado/stack_context.py

```diff
@@ -0,0 +1,70 @@
+from tornado.stack_context import StackContextInconsistentError, _state
+
+from ...context import Context
+
+
+class TracerStackContext(object):
```

Comment:
This can be simplified inheriting the default ``StackContext`` (check different implementations in previous Tornado versions)